scripts/api.js

```diff
@@ -34,6 +30,30 @@ for (let key in httpDependency) {
     if (typeof httpDependency[key] === 'function') httpService[key] = createWrapperFunction(httpDependency[key]);
 }
 
+/****************************************************
+Helpers
+****************************************************/
+
+exports.messages = {};
+
+/**
+ * Sends an HTTP GET request to the specified URL with the provided HTTP options.
+ *
+ * @param {string} to       - The phone number the SMS is sent to.
+ * @param {string} from     - The phone number the SMS is sent from.
+ * @param {string} body     - The actual SMS content.
+ * @return {object}         - The response of the POST request.
+ */
+
+exports.messages.sendSMS = function(to, from, body) {
+    const accountSid = pkg.twilio.utils.getConfiguration("accountSid");
```

Comment:
Avoid use others scripts. You can acces by its private method:

```suggestion
    const accountSid = config.get("accountSid");
```